src/EventManager.ts

```diff
@@ -153,37 +156,38 @@ export class EventManager {
 	 *   event.holding && event.set({x: 10});
 	 * });
 	 */
-	triggerChange(pos: Axis, input: IInputType = null, event = null, holding = false) {
-		const moveTo = this.axm.moveTo(pos);
+	triggerChange(pos: Axis, option: ChangeEventOption = null, holding: boolean = false) {
+		const eventInfo = this.am.getEventInfo();
+		const moveTo = this.am.axm.moveTo(pos);
+		const inputEvent = option && option.event || eventInfo && eventInfo.event || null;
 		const param = {
 			pos: moveTo.pos,
 			delta: moveTo.delta,
-			holding,
-			inputEvent: event,
-			isTrusted: !!event,
-			input,
+			holding: holding,
```

Comment:
Is it different with previous?
previous
```js
//...
delta: moveTo.delta,
holding, //<-- this one
// ...
```

Reply:
the holding property should control in InputObserver.

Reply:
you're right! I fit it!